### DISCUSSION TITLE
isNextDisabled, isPrevDisabled, nextStep, prevStep v-slots not working for Stepper

### DISCUSSION THREAD

**Discussion by PMolnes** [unovue/shadcn-vue#766] [Q&A]
I am trying to use v-slots in the Stepper just like the example in the link below. However, I am just getting the error:
```
Property 'isPrevDisabled' does not exist on type '{ modelValue: number | undefined; }'
```

What could be the reason for this? I have copied the exact example from this link and its not working.

https://github.com/radix-vue/shadcn-vue/blob/7c9aac46a1b3db669daf782c6803fc62c8a10219/apps/www/src/lib/registry/default/example/StepperForm.vue

**Comment by sadeghbarati** [comment]
Share your copied `shadcn-vue` source maybe it has an error
